simplified-main/src/main/res/layout/main_tabbed_host.xml

```diff
@@ -27,6 +27,7 @@
     android:id="@+id/bottomNavigator"
     android:layout_width="0dp"
     android:layout_height="64dp"
+    android:background="?android:attr/windowBackground"
```

Comment:
I'm not sure `windowBackground` is the right thing to use here. Typically this will just be transparent. Can you point me to some documentation that suggests we should set this attribute on the `BottomNavigationView`?

Reply:
Unfortunately, I don't have a good answer to this. I looked at various posts online and this seemed to be what people were using. I tried various colors and this was the only one I could get to work correctly.

Reply:
I should point out actually that you can see this problem in the UI designer if you set the theme to one of our app themes. The bottom navigator appears to be black if you don't set the color. If you set the bottom navigator color to red, you'll get a red bar with black borders. If you set the bottom navigator to the window background, you'll get a transparent view without borders. Frankly, I'd had enough of going down the rabbit hole at that point. :laughing:

Reply:
Yeah, this seems fine for now.

I see we set a value for windowBackground in styles. Historically, you could set an image for windowBackground (like a splash image) and it would display while your application is spinning up before your first activity's onCreate method is called. I don't think this technique is used so much anymore but it was a thing for a while.

I think the root problem was probably that the navigator just didn't have a background set. You sometimes get that black appearance if transparent is set on a view and there's nothing behind it.

Reply:
Only in Android could _white_ + _transparent_ = _black_...